src/lazy/encoder/writer.rs

```diff
@@ -69,7 +69,7 @@ impl<E: Encoding, Output: Write> Writer<E, Output> {
         // TODO: LazyEncoder should define a method to construct a new symtab and/or macro table
         let ion_version = E::ion_version();
         let symbol_table = SymbolTable::new(ion_version);
-        let macro_table = MacroTable::new();
+        let macro_table = MacroTable::with_system_macros();
```

Comment:
🗺️ `MacroTable` now has a `with_system_macros()` method (that pre-populates it with system macro aliases) and an `empty()` method that constructs a new instance with no macros at all. The latter is used for processing encoding directives.